.github/SUPPORT.md

```diff
@@ -11,6 +11,7 @@ For issues with code (and especially if you need to share debug output) we recom
 
  - [Singularity Issues](https://github.com/singularityware/singularity/issues): is recommended for most issues with the Singularity software.
  - [Singularity Hub Issues](https://github.com/singularityhub/singularityhub.github.io/issues): is the board for issues relevant to Singularity Hub.
+ - [Singularity Registry Issues](https://github.com/singularityhub/sregistry/issues): is the board for issues relevant to Singularity Registry
```

Comment:
Should really be Singularity Hub right?  There may be many Singularity Registries but there is only shub.

Reply:
No, lol. Singularity Registry is a different thing :)

https://github.com/singularityhub/sregistry

it's even peer reviewed / published!

http://joss.theoj.org/papers/050362b7e7691d2a5d0ebed8251bc01e

it's times like these the stuff I do feels... totally unnoticed lol.

Reply:
Oh sorry.  I thought shub was a special case of a Singularity registry.  Nevermind.

Reply:
It's like homo sapiens and Neanderthal. Maybe some wayward, back in the day common elements (mostly UI) but different beasties under it all!

Reply:
but yeah Shub (which came first) does deliver the common API endpoints of a registry so Shub works like one too. It's the mamma registry! The HUB.